### PR TITLE
ticdc: update v4.0.3 changes

### PR DESCRIPTION
**Pull request by TomShawn**
### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/3967
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Have version specific changes 
- [ ] Might cause conflicts

### CODE REVIEW

File: ticdc/manage-ticdc.md [hunk 1]
```diff
@@ -105,10 +112,17 @@ Execute the following commands to create a replication task:
 
 ```shell
 cdc cli changefeed create --pd=http://10.0.10.25:2379 --sink-uri="mysql://root:123456@127.0.0.1:3306/"
```
Comment (ran-huang) [inline ticdc/manage-ticdc.md:114]:
```suggestion
cdc cli changefeed create --pd=http://10.0.10.25:2379 --sink-uri="mysql://root:123456@127.0.0.1:3306/" --changefeed-id="simple-replication-task"
```

File: ticdc/manage-ticdc.md [hunk 2]
```diff
@@ -105,10 +112,17 @@ Execute the following commands to create a replication task:
...
-create changefeed ID: 28c43ffc-2316-4f4f-a70b-d1a7c59ba79f info {"sink-uri":"mysql://root:123456@127.0.0.1:3306/","opts":{},"create-time":"2020-03-12T22:04:08.103600025+08:00","start-ts":415241823337054209,"target-ts":0,"admin-job-type":0,"config":{"filter-case-sensitive":false,"filter-rules":null,"ignore-txn-start-ts":null}}
 ```
 
-Configure `--sink-uri`  according to the following format. Currently, the scheme supports `mysql`/`tidb`/`kafka`.
+```shell
+--changefeed-id="simple-replication-task"
```
Comment (ran-huang) [inline ticdc/manage-ticdc.md:118]:
```suggestion
```

File: ticdc/manage-ticdc.md (hunk 1 above)
Comment (TomShawn) [reply ticdc/manage-ticdc.md:114]: @overvenus PTAL

File: ticdc/manage-ticdc.md [hunk 3]
```diff
@@ -104,11 +111,17 @@ Execute the following commands to create a replication task:
...
+ID: simple-replication-task
+Info: {"sink-uri":"mysql://root:123456@127.0.0.1:3306/","opts":{},"create-time":"2020-03-12T22:04:08.103600025+08:00","start-ts":415241823337054209,"target-ts":0,"admin-job-type":0,"sort-engine":"memory","sort-dir":".","config":{"case-sensitive":true,"filter":{"rules":["*.*"],"ignore-txn-start-ts":null,"ddl-allow-list":null},"mounter":{"worker-num":16},"sink":{"dispatchers":null,"protocol":"default"},"cyclic-replication":{"enable":false,"replica-id":0,"filter-replica-ids":null,"id-buckets":0,"sync-ddl":false},"scheduler":{"type":"table-number","polling-time":-1}},"state":"normal","history":null,"error":null}
 ```
 
-Configure `--sink-uri`  according to the following format. Currently, the scheme supports `mysql`/`tidb`/`kafka`.
+- `--changefeed-id`: The ID of the replication task. The format must comply with the `^[a-zA-Z0-9]+(\-[a-zA-Z0-9]+)*$` regular expression. If this ID is not specified, TiCDC automatically generates a UUID (the version 4 format) as the ID.
+- `--sink-uri`: The downstream address of the replication task. Configure `--sink-uri`  according to the following format. Currently, the scheme supports `mysql`/`tidb`/`kafka`.
```
Comment (ran-huang) [inline ticdc/manage-ticdc.md:124]:
```suggestion
- `--sink-uri`: The downstream address of the replication task. Configure `--sink-uri` according to the following format. Currently, the scheme supports `mysql`/`tidb`/`kafka`.
```

File: ticdc/manage-ticdc.md [hunk 4]
```diff
@@ -557,7 +587,7 @@ To create a cyclic replication task, take the following steps:
 ### Usage notes
 
 + Before creating the cyclic replication task, you must execute `cdc cli changefeed cyclic create-marktables` to create the mark tables for the cyclic replication.
-+ Tables with cyclic replication enabled only contain the `[a-zA-Z0-9_]` characters.
++ The name of the table with cyclic replication enabled must comply with the `^[a-zA-Z0-9_]+$` regular expression.
```
Comment (overvenus) [inline ticdc/manage-ticdc.md:590]: What about `match`？
```suggestion
+ The name of the table with cyclic replication enabled must match the `^[a-zA-Z0-9_]+$` regular expression.
```

File: ticdc/manage-ticdc.md [hunk 5]
```diff
@@ -104,11 +111,17 @@ Execute the following commands to create a replication task:
...
+```
+
+```shell
+Create changefeed successfully!
+ID: simple-replication-task
+Info: {"sink-uri":"mysql://root:123456@127.0.0.1:3306/","opts":{},"create-time":"2020-03-12T22:04:08.103600025+08:00","start-ts":415241823337054209,"target-ts":0,"admin-job-type":0,"sort-engine":"memory","sort-dir":".","config":{"case-sensitive":true,"filter":{"rules":["*.*"],"ignore-txn-start-ts":null,"ddl-allow-list":null},"mounter":{"worker-num":16},"sink":{"dispatchers":null,"protocol":"default"},"cyclic-replication":{"enable":false,"replica-id":0,"filter-replica-ids":null,"id-buckets":0,"sync-ddl":false},"scheduler":{"type":"table-number","polling-time":-1}},"state":"normal","history":null,"error":null}
 ```
 
-Configure `--sink-uri`  according to the following format. Currently, the scheme supports `mysql`/`tidb`/`kafka`.
+- `--changefeed-id`: The ID of the replication task. The format must comply with the `^[a-zA-Z0-9]+(\-[a-zA-Z0-9]+)*$` regular expression. If this ID is not specified, TiCDC automatically generates a UUID (the version 4 format) as the ID.
```
Comment (overvenus) [inline ticdc/manage-ticdc.md:123]:
```suggestion
- `--changefeed-id`: The ID of the replication task. The format must match the `^[a-zA-Z0-9]+(\-[a-zA-Z0-9]+)*$` regular expression. If this ID is not specified, TiCDC automatically generates a UUID (the version 4 format) as the ID.
```

**Review by overvenus — COMMENTED** [review]
Rest LGTM

**Pull request merged**